src/recordpage.cpp

```diff
@@ -378,7 +393,7 @@ RecordPage::RecordPage(QWidget* parent)
     connect(stopRecordingShortcut, &QShortcut::activated, this,
             [this] { ui->startRecordingButton->setChecked(false); });
 
-    auto* startRecordingShortcut = new QShortcut(Qt::Key_Return, this);
+    auto* startRecordingShortcut = new QShortcut(Qt::CTRL + Qt::Key_Return, this);
```

Comment:
this change of the shortcut should be a separate commit with an explanation of why you change it